woocommerce-services.php

```diff
@@ -1614,9 +1614,9 @@ public function enqueue_wc_connect_script( $root_view, $extra_args = array() ) {
 			$escaped_arguments = function_exists( 'wc_esc_json' ) ? wc_esc_json( $encoded_arguments ) : esc_attr( $encoded_arguments );
 
 			?>
-				<div class="wcc-root woocommerce <?php echo esc_attr( $root_view ); ?>" data-args="<?php echo $escaped_arguments; ?>">
+				<div class="wcc-root woocommerce <?php echo esc_attr( $root_view ); ?>" data-args="<?php echo $escaped_arguments; // phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped ?>">
 					<span class="form-troubles" style="opacity: 0">
-						<?php printf( __( 'Section not loading? Visit the <a href="%s">status page</a> for troubleshooting steps.', 'woocommerce-services' ), $debug_page_uri ); ?>
+						<?php printf( esc_html(__( 'Section not loading? Visit the <a href="%s">status page</a> for troubleshooting steps.', 'woocommerce-services' ), $debug_page_uri )); ?>
```

Comment:
No change needed here but you could also do:

```suggestion
						<?php printf( esc_html__( 'Section not loading? Visit the <a href="%s">status page</a> for troubleshooting steps.', 'woocommerce-services' ), $debug_page_uri ); ?>
```